analyses/verbose/sarif_helper.go

```diff
@@ -0,0 +1,153 @@
+package verbose
+
+import (
+	"errors"
+	"fmt"
+	"github.com/fabric8-analytics/cli-tools/analyses/driver"
+	"github.com/owenrumney/go-sarif/models"
+	"github.com/owenrumney/go-sarif/sarif"
+	"github.com/rs/zerolog/log"
+	"io/ioutil"
+	"os"
+	"regexp"
+	"strings"
+)
+
```

Comment:
```suggestion
// RegexDependencyLocator --relevant comment--
```
In Golang, every exported member has a relevant comment